Documentation/prerequisites-for-building.md

```diff
@@ -1,16 +1,16 @@
 The following pre-requisites need to be installed for building the repo:
 
-# Install .NET Core 2.1 SDK
+# Install .NET Core 3.1 SDK
 
-Download .NET Core 2.1 SDK from [https://www.microsoft.com/net/download/core](https://www.microsoft.com/net/download/core)
+Download .NET Core 3.1 SDK from [https://www.microsoft.com/net/download/core](https://www.microsoft.com/net/download/core)
 
 You should now be able to use the `dotnet` commands of the CLI tools.
 
 # Windows (Windows 7+)
 
-1. Install [Visual Studio 2017](https://visualstudio.microsoft.com/vs/community/) or [Visual Studio 2019 Preview](https://visualstudio.microsoft.com/vs/preview/), including Visual C++ support.
+1. Install [Visual Studio 2019](https://visualstudio.microsoft.com/vs/preview/), including Visual C++ support.
```

Comment:
This can point to `https://visualstudio.microsoft.com/vs/community/`. No need to force people to previews.